app/controllers/katello/api/v2/capsules_controller.rb

```diff
@@ -4,16 +4,16 @@ class Api::V2::CapsulesController < ::Api::V2::SmartProxiesController
       api_base_url "/katello/api"
     end
 
-    api :GET, '/capsules', 'List all capsules'
+    api :GET, '/capsules', 'List all smart proxies'
```

Comment:
this isn't exactly correct (nor was it before).  This really only lists smart proxies that 'have content'.